app/javascript/flavours/glitch/components/avatar.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { autoPlayGif } from 'flavours/glitch/initial_state';
+import { useHovering } from 'hooks/useHovering';
+import type { Account } from 'types/resources';
+
+type Props = {
+  account: Account;
+  className: string;
+  size: number;
```

Comment:
Only `size` is required.
`account` is treated as required, but can be undefined if an account is suspended.